packages/pancake-profile-sdk/src/profile-sdk.ts

```diff
@@ -0,0 +1,179 @@
+import Web3 from "web3";
+import { Contract } from "web3-eth-contract";
+import { request, gql } from "graphql-request";
+import Cookies from "js-cookie";
+import web3NoAccount from "./utils/web3";
+import { getProfileContract } from "./utils/contractHelpers";
+import { profileApi, profileSubgraphApi } from "./constants/common";
+import { campaignMap } from "./constants/campaings";
+import teamsList from "./constants/teams";
+import { Achievement, Team, GetProfileResponse, Profile, Nft } from "./types";
+import { getAchievementDescription, getAchievementTitle, transformProfileResponse } from "./utils/transformHelpers";
+import { getNftByTokenId } from "./utils/collectibles";
+
+type SdkConstructorArguments = {
+  web3?: Web3;
+  chainId?: number;
+};
+
+// move to types
```

Comment:
Move to types?

Reply:
yeah I should go through comments, might be more of this reminders here and there 👍